okteto.yml

```diff
@@ -0,0 +1,14 @@
+name: id
+command: bash
+workdir: /go/src/github.com/replicatedhq/kots-lint
+sync:
+  - .:/go/src/github.com/replicatedhq/kots-lint
+resources:
+  limits:
+    cpu: "2"
+    memory: 4Gi
+externalVolumes:
+  - dev-cache:yarn-cache:/cache/.yarn
+forward:
+  - 2352:2345
```

Comment:
This was a copy/paste job, not sure this is accurate (only needed for debugging?)